package.json

```diff
@@ -23,9 +23,9 @@
   "devDependencies": {
     "@11ty/eleventy": "1.0.1",
     "eleventy-plugin-dart-sass": "1.0.3",
-    "govuk-frontend": "^4.2.0"
+    "govuk-frontend": "^4.7.0"
   },
   "peerDependencies": {
-    "govuk-frontend": "4.x"
+    "govuk-frontend": "4.7.x"
```

Comment:
Lets keep this as 4.x since your changes are backwards compatible with all previous releases and this change would start throwing annoying warnings for people who are not currently at the latest build.

Reply:
Hi Nick. I just checked with both an `npm i` to use the package.json and an `npm ci` to pull in via package-lock.json and in both cases I can confirm that I don't get the card parts of the summary-list package. Checking [govuk-frontend](https://github.com/alphagov/govuk-frontend) it looks like the cards were added in 4.5.0.

As I see it we have two options: we could partially upgrade to 4.5 or go the full way to 4.7. I'm not clear on what benefits there might be going to 4.5 but there might be some I hadn't thought of. Without one of these changes, though, I don't see how the new cards will function.

If this will just throw warnings as you say then we could just keep things as they are and move the version number with a minor increment as I have. However, this does mean that for people on the old package the new stylings would apply incorrectly. I guess technically that's not a breaking change to existing functionality though, even for a styling library.

Reply:
Would need to check myself but this surely is some sort of caching on your machine since a fresh install with these dependencies should get you the latest build?

Reply:
I recloned the repo and ran a fresh install, and deleted the node modules between running `npm i` and `npm ci` to boot. In addition, I have never run an install prior to 4.x being the latest version so I don't see how it could have cached. I see what you mean though in that you'd expect it to grab the latest version. I'll test with the carat see if that helps.

Reply:
Actually, scratch that as it'll have the devDeps on my machine, not the peerDeps. So I suppose in actual fact the 4.x in peerDeps can stay and it just needs to the ^4.2.0 in the devDeps upgraded.

Reply:
Ok, I have checked by pulling into a new project and it does indeed pull through the right deps that way. Have switched to 4.x as suggested.